core/src/main/scala/org/http4s/MimeDB.scala

```diff
@@ -5,9 +5,9 @@ private[http4s] trait MimeDB {
   lazy val allMediaTypes
     : List[MediaType] = Nil ++ x_shader.all ++ x_conference.all ++ video.all ++ text.all ++ multipart.all ++ model.all ++ message.all ++ image.all ++ font.all ++ chemical.all ++ audio.all ++ application.all
   val Compressible: Boolean = true
-  val Uncompressible: Boolean = true
+  val Uncompressible: Boolean = false
```

Comment:
I guess maybe I thought these were sum types instead of boolean constants.  Would stronger types help here?

Reply:
Maybe?  Booleans are a smell.  I think either way, this fixes a bug and should be merged.